modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -1591,6 +1594,176 @@ TEST_P(Test_ONNX_layers, CumSum)
     testONNXModels("cumsum_3d_dim_2");
 }
 
+struct TestCaseInput
+{
+    std::vector<std::string> input_paths;
+    std::vector<std::string> output_paths;
+    std::string model_path;
+    std::string name;
+};
+
+std::ostream& operator<<(std::ostream& os, const TestCaseInput& test_case)
+{
+    return os << test_case.name;
```

Comment:
Under test case list I mean something like this as a part of the tests source code: https://github.com/opencv/opencv/blame/4.5.4/modules/dnn/perf/perf_convolution.cpp#L15-L36